<?xml version="1.0" encoding="utf-8"?>

<launch>

    <include file="$(find drone_controller)/launch/main_simulation.launch" />

    <node pkg="drone_controller" name="program_controller_node" type="person_scenario_controller.py" output="screen">
    </node>

    <node pkg="drone_controller" name="person_navigation_node" type="person_navigation.py" output="screen">
        <param name="gains/p" type="double" value="1" />
        <param name="gains/i" type="double" value="0" />
        <param name="gains/d" type="double" value="0.75" />
    </node> 

    <include file="$(find darknet_ros)/launch/darknet_ros.launch">
        <arg name="image" default="/mixer/sensors/camera" />
        <arg name="network_param_file" default="$(find darknet_ros)/config/yolov2-tiny.yaml"/>
    </include>

    <node pkg="rqt_image_view" type="rqt_image_view" name="darknet_view" args="/darknet_ros/detection_image" output="screen"/>
    
</launch>
